win32ss/include/ntuser.h

```diff
@@ -195,7 +195,7 @@ typedef struct tagIMC
     HWND           hImeWnd;
 } IMC, *PIMC;
 
-#ifndef _WIN64
+#if !defined(_WIN64) && !defined(__cplusplus)
```

Comment:
Why "not defined(__cplusplus)" ? What happens if you keep the original ifndef?

Reply:
In C++, I defined `NONAMELESSUNION` before `#include`.
So, in this case, macro `DUMMYSTRUCTNAME` is not empty.

Reply:
Reverted.